Vagrantfile

```diff
@@ -46,12 +47,13 @@ Vagrant.configure(2) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder ".", "/vagrant",
+  config.vm.synced_folder ".", "/home/nico/",
```

Comment:
sorry this slipped through, was done as a test.. as was still getting the hang of vagrant :)
I will revert and sync to the vagrant folder